packages/debug/src/node/vscode/vscode-debug-adapter-contribution.ts

```diff
@@ -98,13 +98,11 @@ export abstract class AbstractVSCodeDebugAdapterContribution implements DebugAda
     }
 
     protected async parse(): Promise<VSCodeExtensionPackage> {
-        let text = (await fs.readFile(this.pckPath)).toString();
+        let text: string = await fs.readFile(this.pckPath, 'utf8');
```

Comment:
Nitpick: The `string` type is not necessary here, TS should infer that.

Reply:
It is to enforce `readFile` to return `string`. i.e. if I remove the `utf8` argument it returns `Buffer`...  I usually prefer to let TypeScript infer too, except when a function has overloads with incompatible return types like this. Tell me if you'd still prefer to just rely on inference.

Reply:
Right, but as long as the `encoding` argument is there, `readFile` will always be enforced to return `string`, right? At least that's what the [typescript definition](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1fc1033f3a50e169b088729cbd691f60108c0262/types/fs-extra/index.d.ts#L217) says:

```ts
export function readFile(file: PathLike | number, encoding: string): Promise<string>;
```

So I don't see a reason to keep the `string` type for `text` there.